packages/tracing-state-updater/src/services/operationsService.ts

```diff
@@ -29,7 +29,7 @@ export const operationsServiceBuilder = (
             state: data.state,
           },
           {
-            headers: { ...correlationIdToHeader(uuidv4()) },
+            headers: { ...correlationIdToHeader(uuidv4()) }, // Temporary value to valorize with correlationId implementation
```

Comment:
is this addressed in some other PR/task?

Reply:
Resolved in #20